test/db/formats/pdb

```diff
@@ -72,6 +75,7 @@ CMDS=!rz-bin -P ${RZ_FILE} | grep -ao "struct CTest2"
 EXPECT=<<EOF
 struct CTest2
 EOF
```

Comment:
For classes broken tests could you please add a link to the https://github.com/rizinorg/rizin/issues/1205

Reply:
This is still BROKEN

Reply:
This is LF_CLASS.